web-frontend/src/pages/TransferCoins/SendCoins/SendConfirmation.tsx

```diff
@@ -17,6 +18,7 @@ import ToolTip from './ToolTip';
 export function SendConfirmation({ ...props }) {
   const navigate = useNavigate();
   const { amount, nickname, recipient, valid, fees, setValid } = props;
+  const formattedRecipientAddress = maskAddress(recipient);
   const reversedAmount = reverseFormatStandard(amount);
   const amountInNanoMAS = fromMAS(reversedAmount).toString();
   const total = +amountInNanoMAS + +fees;
```

Comment:
For what we have `+` before `amountInNanoMAS` ? And for what we have `+ +` after ? 🤔

Reply:
it's a parseint

Reply:
Okay